### DISCUSSION TITLE
Do not send telemetry for frame_from_hyper_query

### DISCUSSION THREAD

**Discussion by Niranjanpoudelusu** [innobi/pantab#379] [Q&A] [answered]
If I understand correctly, when directly calling ```frame_from_hyper_query```  the value of ```process_param``` should be  ```{"log_config" : ""}``` to avoid sending the telemetry data right?

**Comment by WillAyd** [comment, marked as answer]
pantab does not send telemetry at all. It is hard-coded into the implementation. See 

https://github.com/innobi/pantab/blob/9a18d6cdded68758a0a72bd27c444de87e3a28b1/src/pantab/reader.cpp#L445

and 

https://github.com/innobi/pantab/blob/9a18d6cdded68758a0a72bd27c444de87e3a28b1/src/pantab/writer.cpp#L742
